docs/data/material/components/table/EnhancedTable.tsx

```diff
@@ -253,21 +257,44 @@ function EnhancedTableToolbar(props: EnhancedTableToolbarProps) {
 }
 
 export default function EnhancedTable() {
-  const [order, setOrder] = React.useState<Order>('asc');
-  const [orderBy, setOrderBy] = React.useState<keyof Data>('calories');
+  const [order, setOrder] = React.useState<Order>(DEFAULT_ORDER);
+  const [orderBy, setOrderBy] = React.useState<keyof Data>(DEFAULT_ORDER_BY);
   const [selected, setSelected] = React.useState<readonly string[]>([]);
   const [page, setPage] = React.useState(0);
   const [dense, setDense] = React.useState(false);
-  const [rowsPerPage, setRowsPerPage] = React.useState(5);
-
-  const handleRequestSort = (
-    event: React.MouseEvent<unknown>,
-    property: keyof Data,
-  ) => {
-    const isAsc = orderBy === property && order === 'asc';
-    setOrder(isAsc ? 'desc' : 'asc');
-    setOrderBy(property);
-  };
+  const [visibleRows, setVisibleRows] = React.useState<Data[] | null>(null);
+  const [rowsPerPage, setRowsPerPage] = React.useState(DEFAULT_ROWS_PER_PAGE);
+  const [emptyRowsHeight, setEmptyRowsHeight] = React.useState(0);
+
+  React.useEffect(() => {
+    let rowsOnMount = stableSort(
+      rows,
+      getComparator(DEFAULT_ORDER, DEFAULT_ORDER_BY),
+    );
+    rowsOnMount = rows.slice(
```

Comment:
I think this should be:
```suggestion
    rowsOnMount = rowsOnMount.slice(
```
on initial mount, because our visual regression test detected a difference (see https://app.argos-ci.com/mui/material-ui/builds/11398/41761580). On initial mount, the calorie column should be sorted in ascending order which does not happen in https://deploy-preview-33236--material-ui.netlify.app/material-ui/react-table/#sorting-amp-selecting.